pkgs/development/tools/casperjs/default.nix

```diff
@@ -28,9 +28,9 @@ in stdenv.mkDerivation rec {
     make test
   '';
 
-
   installPhase = ''
-    mv $PWD $out
+    mkdir $out
```

Comment:
Should work, but I like `mv` because it avoids creating unnecessary copies. Is `cp` preferred somehow?